### ISSUE TITLE
can create a user with an empty password - password required to log in

### ISSUE THREAD

**Issue by msmith125** [CodeForBaltimore/Bmore-Responsive#173]
**Describe the bug**
A user can be created with an empty password but you cannot log into HealthCare RollCall UI with an empty password

**To Reproduce**
1. Create a user using Post url/user with empty password ""

**Expected behavior**
A password should probably be required and the user should not be created without a password.

**Comment by revjtanton** [comment]
This will need a followup ticket in Healthcare Rollcall to inform the user of the new restrictions.
- minimum of 8 characters
- 1 uppercase
- 1 lowercase
- 1 special
- 1 digit

**Comment by msmith125** [comment]
I tested this out, without a password and with donuts as a password.  Error 422 returned, which only indicates invalid input.  When I changed the password to contain 8 char, 1 upper, 1 lower, 1 special and 1 digit, I was able to create a new user.  I can enter a new story based on this criteria.  I think it would be helpful if the error message reflected the password requirements and if the swagger doc reflected the password requirements.  Not sure if a something other than the 422 could tell the user creating the new user that?
